docs/developer-docs/backend/rust/11-stable-structures.md

```diff
@@ -0,0 +1,13 @@
+# 11: Stable structures
```

Comment:
Is this going to be expanded with a full tutorial at some point? If not it may be worth removing it in favor of a section in `7-upgrading.md`.

Reply:
Originally this included the full content from the stable structures README, but in a previous suggestion by @roman-kashitsyn , it was decided to link to the stable structures repo instead since the documentation there is expected to change frequently.